src/app/common/bitcoin-utils.ts

```diff
@@ -0,0 +1,10 @@
+import BigNumber from 'bignumber.js';
+
+import { BTC_DECIMALS } from '@shared/constants';
+
+import { initBigNumber } from './utils';
+
+export const satToBtc = (sat: number | string | BigNumber) => {
+  const satBigNumber = initBigNumber(sat);
+  return satBigNumber.shiftedBy(-BTC_DECIMALS);
+};
```

Comment:
Maybe we could create a reusable factory fn that takes a decimals and returns this function. Would be useful for where it's used with STX too, something like

```ts
function fractionalUnitToUnit(decimals: number) {
  return (unit: number | string | BigNumber) => {
    const satBigNumber = initBigNumber(unit);
    return satBigNumber.shiftedBy(-decimals);
  }
}
const satToBtc = fractionalUnitToUnit(BTC_DECIMALS);
const microStxToStx = fractionalUnitToUnit(STX_DECIMALS)
```

Taking the time to refactor existing stx unit conversation functions would be worthwhile to reuse logic we already have. 

Also, I reckon we should consider switching how we divide btc/stx logic. This PR uses `src/app/common/bitcoin-utils.ts` because there's an equivalent `stacks-utils.ts`. Blockchain → Functions.

This approach makes it harder to reuse logic that's shared between functions, like this example, and spreads out very similar logic to different parts of the codebase. What if we switch to more: Functions → Blockchain grouping (or just `./functions` alone if there isn't much logic), where this might become housed in `./money/unit-conversion` exporting these methods? Thoughts?

Reply:
Yep, sounds like a good idea. I'll can move them.